Objective-C/Internal/Replicator/CBLWebSocket.mm

```diff
@@ -467,10 +470,11 @@ - (void)URLSession:(NSURLSession *)session writeClosedForStreamTask:(NSURLSessio
 }
 
 - (void) streamClosed: (BOOL)isWrite {
+    BOOL expectedClose = _requestedClose || _closeOnError;
     CBLLog(WebSocket, @"CBLWebSocket %s stream closed%s",
            (isWrite ? "write" : "read"),
-           (_requestedClose ? "" : " unexpectedly"));
-    if (!_requestedClose) {
+           (expectedClose ? "" : " unexpectedly"));
```

Comment:
It'll get reported later in a log message starting `CBLWebSocket CLOSED WITH ERROR`.